lib/chai/core/assertions.js

```diff
@@ -672,6 +672,79 @@ Assertion.addProperty('true', function () {
   );
 });
 
+/**
+ * TODO
+ */
+Assertion.addProperty('callable', function () {
+  const val = flag(this, 'object')
+  const ssfi = flag(this, 'ssfi')
+  const message = flag(this, 'message')
+  const msg = message ? `${message}: ` : ''
+  const negate = flag(this, 'negate');
+
+  const assertionMessage = negate ?
+    `${msg}expected ${_.inspect(val)} not to be a callable function` : 
+    `${msg}expected ${_.inspect(val)} to be a callable function`;
+
+  const isCallable = !['Function', 'AsyncFunction', 'GeneratorFunction', 'AsyncGeneratorFunction'].includes(_.type(val));
```

Comment:
is this meant to be negated? shouldn't it be true when it includes the type?

Reply:
Yess! Good catch.